lib/katello/plugin.rb

```diff
@@ -123,7 +123,7 @@
        :content_hosts,
        :caption => N_('Content Hosts'),
        :url => '/content_hosts',
-       :url_hash => {:controller => 'katello/api/v2/systems',
+       :url_hash => {:controller => 'api/v2/hosts',
```

Comment:
@jlsherrill Out of curiosity; where does this controller live? All others are prefixed with katello. I did some poking around and couldn't find anything.